playbooks/tests/tasks/barbican.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: test barbican
+  hosts: controller[0]
+  tasks:
+  - name: OpenStack service list should include barbican
+    shell: . /root/stackrc; openstack service list | grep barbican
+
+  - name: OpenStack endpoint list should include barbican
+    shell: . /root/stackrc; openstack endpoint list | grep barbican
+
+  - name: Create a barbican secret
+    shell: >
+      . /root/stackrc;
+      openstack secret store --secret-type passphrase --name 'passphrase' --payload 'Pa$$phrasE' -c "Secret href" -f value
+    register: secret_ref
+
+  - name: List barbican secrets
+    shell: . /root/stackrc; openstack secret list
```

Comment:
we should verify list show's secret created above.

Also, show add another test show - for secret created.